### PR TITLE
Math-Magicians (Add components)

### PR DESCRIPTION
**Pull request by rahimialijan**
# Math Magician Project: Add Calculator as Component

- Fixed the linter errors
- Add component for react project
- Add CSS style for all elements

### CODE REVIEW

File: src/components/Calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <div className="btn light-gray">
+      +/-
+    </div>
+    <div className="btn light-gray">
+      %
+    </div>
+    <div className="btn orange">
+      /
+    </div>
+    <div className="btn">
```
Comment (AmaduKamara) [inline src/components/Calculator.jsx]: - [ ] According to the project requirements, the Calculator.js component should have at least one child component. Kindly create a separate component that you will use inside this component as a child component. 👍🏽

Screenshot 2023-06-06 at 11 38 25 AM

File: src/App.css [hunk 2]
```diff
@@ -1,39 +1,87 @@
...
+.screen {
+  grid-column: 1/-1;
   color: white;
+  text-align: center;
+  padding: 0.5rem;
+  font-size: 4rem;
+  overflow: hidden;
+  background-color: rgba(43, 40, 40, 0.993);
+  border-radius: 4px;
 }
```
Comment (AmaduKamara) [inline src/App.css]: - [ ] Kindly style your calculator app to match the original design given. I would say your design looks nice but let's try to follow project requirements as much as we can. Please modify your styles 👍🏽


**Your Design**

Screenshot 2023-06-06 at 11 40 14 AM


**The Original Design**

Screenshot 2023-06-06 at 11 40 35 AM

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @rahimialijan,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ Created Calculator components as required 👍🏽
✅ The app runs fine 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by ShadyShawkat — APPROVED** [review]
## Hi @rahimialijan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**